src/Lagrangian/levelmethod.jl

```diff
@@ -85,7 +85,7 @@ function lagrangian_method!{S,T}(lp::LinearProgramData{LevelMethod{S,T}}, m::JuM
     while iteration < levelmethod.maxit
         iteration += 1
         # Evaluate the real function and a subgradient
-        # m.internalModelLoaded = false
+        m.internalModelLoaded = false # need to do this smarter...
```

Comment:
I forgot about this... there is one model and the objective changes between being linear and quadratic. If we don't want to use the quadratic solver for the linear part of this, we can't keep the quadratic terms.
Normally we would go into 
https://github.com/JuliaOpt/JuMP.jl/blob/master/src/quadexpr.jl#L69